dotnet/sample-apps/aws-sdk/deploy/wrapper/layer_arm64.tf

```diff
@@ -0,0 +1,4 @@
+locals {
+  collector_layer_arns_arm64 = {
```

Comment:
collector does not support `arm64`?

Reply:
It does! But if you are asking why all the `layer_arm64.tf` files are basically empty it is because there are no currently publicly published Layer ARNs for the `arm64` layers. This file is used by the release workflow and the canary workflow to test the latest published layers but we can't do that yet until we actually publish them.

When we _do_ release `arm64` layers, we will update this file to have the ARNs of the published layers, like we always do in PRs like #153.